src/BuildASub.java

```diff
@@ -2,30 +2,34 @@
 
 public class BuildASub {
 
-        public static void main(String[] args) throws Exception {
-            String bread = chooseBread();
-            String meat = chooseMeat();
+    public static void main(String[] args) throws Exception {
+        String bread = chooseBread();
+        String meat = chooseMeat();
         String cheese = chooseCheese();
         Boolean isToastRequired = doYouWantToasted();
         String sauce = chooseSauce();
         //initalising the string which will hold the salad order.
         String saladChoice = "";
 
+
+        if (isToastRequired) {
+            System.out.println("Your sub will be toasted");
+        } else {
+            System.out.println("Cold bread for you");
+        }
         //Calling addSalad method to check if they user would like salad or not.
-        if(addSalad())
-        {
+
+        if (addSalad()) {
             //if yes, call the saladChoice method to build the customer's salad order.
             saladChoice = saladChoice();
+            System.out.println("Salad choice: " + saladChoice);
         }//end if
-
         System.out.println("Your chosen bread is: " + bread);
         System.out.println("Your chosen meat is: " + meat);
-        if(isToastRequired){
-            System.out.println("Your sub will be toasted");
-        }else{
-            System.out.println("Cold bread for you");
-        }
-        }
+        System.out.println("Your chosen sauce is " + sauce);
+
```

Comment:
Are we printing an order receipt? If so, we should abstract it out to a printOrder() method. This could take all the options are parameters.